src/watertap_contrib/seto/unit_models/surrogate/tests/test_lt_med_surrogate.py

```diff
@@ -116,8 +117,8 @@ def test_num_effects_domain(self):
         m.fs.liquid_prop = SeawaterParameterBlock()
         m.fs.vapor_prop = WaterParameterBlock()
 
-        error_msg = "invalid value for configuration 'number_effects'"
-        with pytest.raises(ValueError, match=error_msg):
+        error_msg = "The number of effects should be an integer between 3 to 14"
```

Comment:
```suggestion
        error_msg = f"The number of effects was specified as {object_pointing_to_num_effects}. The number of effects should be specified as an integer between 3 to 14."
```

Reply:
Changes made accordingly for both models.